Sources/StatsdClient/StatsdClient.swift

```diff
@@ -240,9 +240,10 @@ private final class StatsdTimer: TimerHandler, Equatable {
 private final class Client {
     private let eventLoopGroupProvider: StatsdClient.EventLoopGroupProvider
     private let eventLoopGroup: EventLoopGroup
-    private let isShutdown = Atomic<Bool>(value: false)
+
     private let address: SocketAddress
-    private var channel: AtomicBox<Box<Channel?>> = AtomicBox(value: Box(nil))
+
+    let isShutdown = NIOAtomic<Bool>.makeAtomic(value: false)
```

Comment:
Did this become `internal` for a specific reason?

Reply:
good catch thanks